internal/endpoints/payloads_test.go

```diff
@@ -315,5 +342,53 @@ var _ = Describe("RequestIdPayloads", func() {
 				Expect(respData.Durations["puptoo:undefined"]).To(Equal("00:00:09.970000"))
 			})
 		})
+
+		reqIdPayloads = getFourReqIdPayloads(requestId, "1")
+		Context("Get to /payloads/{request_id} Verbosity 1", func() {
+			It("should pass the data forward", func() {
+				req, err := makeTestRequest(fmt.Sprintf("/api/v1/payloads/%s", requestId), query)
+				Expect(err).To(BeNil())
+
+				reqIdPayloadData = reqIdPayloads
+				handler.ServeHTTP(rr, req)
+				Expect(rr.Code).To(Equal(200))
+				Expect(rr.Body).ToNot(BeNil())
+
+				var respData structs.PayloadRetrievebyID
+
+				readBody, _ := ioutil.ReadAll(rr.Body)
+				json.Unmarshal(readBody, &respData)
+
+				Expect(respData.Data[0].Service).To(Equal(reqIdPayloads[0].Service))
```

Comment:
I think we should add some null checks for verbosity levels in these tests. Like, we should expect `system_id` to be empty when verbosity is `1` and so on. That'll ensure that we aren't spitting out any extra data by mistake.

Reply:
ah!!  I meant to add that in!  TY I'll go ahead and add that!